templates/ansible/transport.erb

```diff
@@ -12,16 +12,14 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 -%>
-<% if object.kind? -%>
-def fetch_resource(module, link, kind):
+<%=
```

Comment:
The `method_decl` and `method_call` functions were designed to get away from this anti-pattern.